src/services/UseMarketAPI.js

```diff
@@ -58,7 +52,6 @@ export async function fetchCryptoPriceById(params) {
       return null;
     }
   } catch (error) {
-    console.error(`Error fetching price for '${cryptoId}':`, error.message);
-    return null;
+    throw error;
```

Comment:
nit-picking:

```suggestion
    throw new Error("Error fetching crypto price by id: " + error.message);
```

Would like to keep consistent with `src/services/UseUserAPI.js` on how we handle error